modules/lib/src/main/resources/assets/js/app/wizard/ContentWizardPanel.ts

```diff
@@ -1234,9 +1229,29 @@ export class ContentWizardPanel
         this.settingsWizardStep = new ContentWizardStep(i18n('field.settings'), this.settingsWizardStepForm, 'icon-wrench');
         steps.push(this.settingsWizardStep);
 
+        const iconClass: string = this.canEveryoneRead(this.getPersistedItem()) ? 'icon-unlock' : 'icon-lock';
+        this.editPermissionsToolbarButton = new ContentWizardStep(i18n('field.access'), this.settingsWizardStepForm, iconClass);
+        this.editPermissionsToolbarButton.getTabBarItem().addClass('edit-permissions-button');
+        this.editPermissionsToolbarButton.getTabBarItem().onClicked(this.handleEditPermissionsButtonClicked.bind(this));
+        steps.push(this.editPermissionsToolbarButton);
+
+        this.addAccessibilityToSteps(steps);
+
         return steps;
     }
 
+    private addAccessibilityToSteps(steps: ContentWizardStep[]): void {
+        steps.forEach(step => this.addAccessibilityToAStep(step));
```

Comment:
Almost there:
`(step: ContentWizardStep) => this.addAccessibilityToAStep(step)`